.pre-commit-config.yaml

```diff
@@ -14,3 +14,5 @@ repos:
         exclude: "{{ cookiecutter.package_name }}"
       - id: debug-statements
         exclude: "{{ cookiecutter.package_name }}"
+ci:
+  autofix_prs: false
```

Comment:
Do we want to add the auto update?

Reply:
It's on by default, so this should be fine?